services/frontend-v3/src/redux/slices/errorsSlice.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from "@reduxjs/toolkit";
+
+const errorsSlice = createSlice({
+  name: "errors",
+  initialState: [],
+  reducers: {
+    addError(state, action) {
+      const error = action.payload;
+      let serializedError;
+      if (error.isAxiosError) {
+        serializedError = {
+          title: error.config.method.toUpperCase() + " " + error.config.url,
+          isAxiosError: true,
+          description: [
+            error.response && error.response.data
+              ? [error.response.status, error.response.statusText]
+              : ["No response from backend"],
+          ],
+        };
+      } else {
+        serializedError = {
+          title: error.message,
+          isAxiosError: false,
+          description: error.stack.split("\n"),
+        };
+      }
+
+      state = state.push(serializedError);
```

Comment:
Something cool with redux toolkit, the state can be mutated, https://redux-toolkit.js.org/api/createReducer#direct-state-mutation,  and the line can be simplified by doing this
```suggestion
      state.push(serializedError);
```